### ISSUE TITLE
Can't import file from the same directory

### ISSUE THREAD

**Issue by ok3721** [microsoft/pylance-release#4861]
<h3> Environment data </h3>

-   Language Server version: 2023.9.20
-   OS and version: win32 x64
-   Python version (and distribution if applicable, e.g. Anaconda): 3.10
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: off

<h3> Code Snippet </h3>

File structure:
project/
├── main.py
└── module1.py

```python
import module1
```

<h3> Expected behavior </h3>

No problem detected by pylance

<h3> Actual behavior </h3>

Report problem `Import "module1" could not be resolved Pylance(reportMissingImports)`

This behavior start from version 2023.9.11, can be fixed by revert to version 9.10 or add current directory to `python.analysis.extraPaths`

<h3> Logs </h3>

<details>

```
2023-09-21 16:50:32.248 [info] [Info  - 4:50:32 PM] (4388) Pylance language server 2023.9.20 (pyright 35eaeca4) starting
2023-09-21 16:50:32.249 [info] [Info  - 4:50:32 PM] (4388) Server root directory: C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist
2023-09-21 16:50:32.271 [info] [Info  - 4:50:32 PM] (4388) Starting service instance "<default>"
2023-09-21 16:50:32.301 [info] [Info  - 4:50:32 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:50:32.302 [info] [Info  - 4:50:32 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:50:32.305 [info] [Warn  - 4:50:32 PM] (4388) stubPath typings is not a valid directory.
2023-09-21 16:50:32.428 [info] [Info  - 4:50:32 PM] (4388) Assuming Python version 3.10
2023-09-21 16:50:32.871 [info] [Info  - 4:50:32 PM] (4388) No source files found.
2023-09-21 16:50:33.292 [info] [Info  - 4:50:33 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:50:33.292 [info] [Info  - 4:50:33 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:50:33.297 [info] [Warn  - 4:50:33 PM] (4388) stubPath typings is not a valid directory.
2023-09-21 16:50:33.425 [info] [Info  - 4:50:33 PM] (4388) Assuming Python version 3.10
2023-09-21 16:50:33.881 [info] [Info  - 4:50:33 PM] (4388) No source files found.
2023-09-21 16:50:39.290 [info] [Info  - 4:50:39 PM] (4388) [BG(1)] Long operation: getSemanticTokens full at D:\ytck_new\new_requtttt_check.py (4433ms)
2023-09-21 16:59:34.806 [info] [Info  - 4:59:34 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:59:34.806 [info] [Info  - 4:59:34 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 16:59:34.806 [info] [Warn  - 4:59:34 PM] (4388) stubPath typings is not a valid directory.
2023-09-21 16:59:35.032 [info] [Info  - 4:59:35 PM] (4388) Assuming Python version 3.10
2023-09-21 16:59:35.545 [info] [Info  - 4:59:35 PM] (4388) No source files found.
2023-09-21 16:59:39.613 [info] [Info  - 4:59:39 PM] (4388) [BG(1)] Long operation: checking: D:\ytck_new\new_requtttt_check.py (2860ms)
2023-09-21 16:59:39.613 [info] [Info  - 4:59:39 PM] (4388) [BG(1)] Long operation: analyzing: D:\ytck_new\new_requtttt_check.py (3995ms)
2023-09-21 17:47:28.171 [info] (4388) No configuration file found.
2023-09-21 17:47:28.171 [info] (4388) No pyproject.toml file found.
2023-09-21 17:47:28.171 [info] [Info  - 5:47:28 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 17:47:28.172 [info] [Info  - 5:47:28 PM] (4388) Setting pythonPath for service "<default>": "C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe"
2023-09-21 17:47:28.172 [info] [Warn  - 5:47:28 PM] (4388) stubPath typings is not a valid directory.
2023-09-21 17:47:28.334 [info] [Info  - 5:47:28 PM] (4388) Assuming Python version 3.10
2023-09-21 17:47:28.335 [info] (4388) Assuming Python platform Windows
2023-09-21 17:47:28.836 [info] [Info  - 5:47:28 PM] (4388) Search paths for .
2023-09-21 17:47:28.837 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\typeshed-fallback\stdlib
2023-09-21 17:47:28.837 [info] [Info  - 5:47:28 PM] (4388)   .
2023-09-21 17:47:28.837 [info] [Info  - 5:47:28 PM] (4388)   typings
2023-09-21 17:47:28.837 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\typeshed-fallback\stubs\...
2023-09-21 17:47:28.837 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\bundled\stubs
2023-09-21 17:47:28.838 [info] [Info  - 5:47:28 PM] (4388)   C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs
2023-09-21 17:47:28.838 [info] [Info  - 5:47:28 PM] (4388)   C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib
2023-09-21 17:47:28.838 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0
2023-09-21 17:47:28.838 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages
2023-09-21 17:47:28.839 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32
2023-09-21 17:47:28.839 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib
2023-09-21 17:47:28.839 [info] [Info  - 5:47:28 PM] (4388)   C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pythonwin
2023-09-21 17:47:28.840 [info] [Info  - 5:47:28 PM] (4388)   C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0
2023-09-21 17:47:28.840 [info] [Info  - 5:47:28 PM] (4388)   C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib\site-packages
2023-09-21 17:47:28.840 [info] [Info  - 5:47:28 PM] (4388) Adding fs watcher for library directories:
 C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs
C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib
C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0
C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages
C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32
C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib
C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pythonwin
C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0
C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib\site-packages
2023-09-21 17:47:28.842 [info] (4388) Searching for source files
2023-09-21 17:47:28.844 [info] [Info  - 5:47:28 PM] (4388) No source files found.
2023-09-21 17:47:28.926 [info] (4388) [IDX(FG)] index libraries . (index) ...
2023-09-21 17:47:28.926 [info] (4388) [IDX(FG)]   read stdlib indices (1ms)
2023-09-21 17:47:28.928 [info] (4388) [IDX(FG)] index libraries . (index) [succeed] (4ms)
2023-09-21 17:47:29.871 [info] (4388) [BG(1)] analyzing: D:\ytck_new\new_requtttt_check.py ...
2023-09-21 17:47:29.871 [info] (4388) [BG(1)]   parsing: D:\ytck_new\new_requtttt_check.py (930ms)
2023-09-21 17:47:29.872 [info] [Info  - 5:47:29 PM] (4388) Could not import 'new_upload' in file 'D:\ytck_new\new_requtttt_check.py'
2023-09-21 17:47:29.873 [info] [Info  - 5:47:29 PM] (4388)   Looking in stubPath 'typings'
2023-09-21 17:47:29.874 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'typings'
2023-09-21 17:47:29.875 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'typings'
2023-09-21 17:47:29.876 [info] [Info  - 5:47:29 PM] (4388)   Looking in root directory of execution environment '.'
2023-09-21 17:47:29.876 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path '.'
2023-09-21 17:47:29.877 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path '.'
2023-09-21 17:47:29.877 [info] [Info  - 5:47:29 PM] (4388)   Finding python search paths
2023-09-21 17:47:29.877 [info] [Info  - 5:47:29 PM] (4388)   Executing interpreter: 'C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\python3.10.exe'
2023-09-21 17:47:29.877 [info] [Info  - 5:47:29 PM] (4388)   Skipping 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\python310.zip' because it is not a valid directory
2023-09-21 17:47:29.877 [info] [Info  - 5:47:29 PM] (4388)   Received 9 paths from interpreter
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib
2023-09-21 17:47:29.878 [info] [Info  - 5:47:29 PM] (4388)     C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\Pythonwin
2023-09-21 17:47:29.879 [info] [Info  - 5:47:29 PM] (4388)     C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0
2023-09-21 17:47:29.879 [info] [Info  - 5:47:29 PM] (4388)     C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\site-packages
2023-09-21 17:47:29.879 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs'
2023-09-21 17:47:29.880 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs'
2023-09-21 17:47:29.880 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\DLLs'
2023-09-21 17:47:29.880 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib'
2023-09-21 17:47:29.880 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib'
2023-09-21 17:47:29.881 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib'
2023-09-21 17:47:29.881 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0'
2023-09-21 17:47:29.881 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0'
2023-09-21 17:47:29.881 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0'
2023-09-21 17:47:29.881 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32'
2023-09-21 17:47:29.882 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\win32\lib'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pythonwin'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pythonwin'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pythonwin'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0'
2023-09-21 17:47:29.883 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0'
2023-09-21 17:47:29.884 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0'
2023-09-21 17:47:29.885 [info] [Info  - 5:47:29 PM] (4388)   Looking in python search path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib\site-packages'
2023-09-21 17:47:29.886 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib\site-packages'
2023-09-21 17:47:29.887 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\Lib\site-packages'
2023-09-21 17:47:29.887 [info] [Info  - 5:47:29 PM] (4388)   Looking in bundled stubs path 'C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\bundled\stubs'
2023-09-21 17:47:29.888 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve stub package using root path 'C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\bundled\stubs'
2023-09-21 17:47:29.889 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\bundled\stubs'
2023-09-21 17:47:29.890 [info] [Info  - 5:47:29 PM] (4388)   Looking for typeshed stdlib path
2023-09-21 17:47:29.890 [info] [Info  - 5:47:29 PM] (4388)   Looking for typeshed stdlib path
2023-09-21 17:47:29.891 [info] [Info  - 5:47:29 PM] (4388)   Attempting to resolve using root path 'C:\Users\AOSS\.vscode\extensions\ms-python.vscode-pylance-2023.9.20\dist\typeshed-fallback\stdlib'
2023-09-21 17:47:29.891 [info] [Info  - 5:47:29 PM] (4388)   Typeshed path not found
2023-09-21 17:47:29.891 [info] [Info  - 5:47:29 PM] (4388)   Looking for typeshed third-party path
2023-09-21 17:47:29.891 [info] [Info  - 5:47:29 PM] (4388)   Looking for typeshed stubs path
2023-09-21 17:47:29.891 [info] [Info  - 5:47:29 PM] (4388)   Typeshed path not found


```

</details>

**Comment by tomtastico** [comment]
Having the same problem, switching to the pre-release version of pylance seems to resolve it

**Comment by DanielNoord** [comment]
I'm also having issues with imports not being found in my `virtualenv`. Downgrading fixed the issue.
Could it be related to https://github.com/microsoft/pyright/issues/5943 and the fix included in 1.1.328?

**Comment by debonte** [comment]
Please upgrade to Pylance 2023.9.21 (our latest prerelease build) which includes Pyright 1.1.328, and see if the issue is fixed for you.

**Comment by rchiodo** [comment]
This repros for me if the file is opened by itself and not in a workspace/folder. @ok3721's logs indicate the <default> workspace, which only happens when opening loose files. 

I'm unsure if this is by design or not though. Without a folder open, there's no default path to use.

**Comment by michalpokusa** [comment]
Seems like `v2023.9.20` changed something, I am also getting error all over the project because unresolved imports.

Going back to `v2023.9.10` fixes the problem.

**Comment by rchiodo** [comment]
There was a bug introduced in 2023.9.20 with relation to symbolic links. 2023.9.21 fixes that issue.

The problem here is default search paths. Not sure how we changed that yet, but I'm not sure why we'd add every folder for every file to the search path list.

**Comment by rchiodo** [comment]
Ah okay we add the '.' path to the list of search paths. That's not resolving because of change introduced in 2023.9.20 to turn everything into a real path. 

So this is not by design and is a bug. We should be able to fix this pretty soon.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2023.9.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023931-27-September-2023-PreRelease)
